validate.sh

```diff
@@ -40,10 +42,18 @@ else
   test $GOFMT_LINES -eq 0 || die "gofmt needs to be run, ${GOFMT_LINES} files have issues.  Below is a list of files to review:\n`gofmt -l *.go pbs adapters`"
 fi
 
+# Run the actual tests. Make sure there's enough coverage too, if the flags call for it.
 if $COVERAGE; then
   ./scripts/check_coverage.sh
+
+  if [ "$RACE" -ne "0" ]; then
```

Comment:
Does the $RACE if condition need to be inside the $COVERAGE if body? It is executed either way.

Reply:
derp.... no. no it doesn't. Just sloppy of me.

Cleaning up now.